sdks/java/core/src/main/java/org/apache/beam/sdk/util/FinishedTriggersBitSet.java

```diff
@@ -64,4 +64,9 @@ public void clearRecursively(ExecutableTrigger trigger) {
   public FinishedTriggersBitSet copy() {
     return new FinishedTriggersBitSet((BitSet) bitSet.clone());
   }
+
+  @Override
+  public String toString() {
+    return "FinishedTriggersBitSet" + bitSet.toString();
```

Comment:
Please do use the standard library, though. `MoreObjects.toStringHelper(this).addValue(bitSet)` is robust to moves, renames, etc. It is just lovely.